manager/notifier.go

```diff
@@ -164,6 +165,38 @@ func (n *notifier) sendPagerDutyNotification(serviceKey string, a *Alert) error
 	return nil
 }
 
+func (n *notifier) sendHipChatNotification(authToken string, roomId int32, color string, notify bool, a *Alert) error {
+	// https://www.hipchat.com/docs/apiv2/method/send_room_notification
+	incidentKey := a.Fingerprint()
+	buf, err := json.Marshal(map[string]interface{}{
+		"color":  color,
+		"message":  a.Description,
+		"notify": notify,
+	})
+	if err != nil {
+		return err
+	}
+
+	resp, err := http.Post(
```

Comment:
I am thinking that the answer is no. It looks like golang does not provide a simple way to do timeouts using the http.Post api. Unless I am mistaken? Would it be good to make a ticket to address this in both this implementation and the pagerduty implementation?